app/static/js/local/gallery_edit.js

```diff
@@ -0,0 +1,26 @@
+CKEDITOR.replace( 'description', {
+    customConfig: '/static/js/ckeditor/custom_config.js'
```

Comment:
Going back to @memo330179's comment could we use a base install say from a CDN and use this to load the custom parts  like the bootstrap theme?